e2e/utils/global-teardown.ts

```diff
@@ -0,0 +1,11 @@
+import { existsSync, rmSync } from 'fs-extra';
+
+export default function () {
+  if (global.nxLocalRegistryProcess) {
+    global.nxLocalRegistryProcess.kill();
+  }
+
+  if (existsSync(global.nxLocalRegistryStorage)) {
+    rmSync(global.nxLocalRegistryStorage);
+  }
```

Comment:
Let's console.log that stuff has been cleaned up

Reply:
added